.tina/config.ts

```diff
@@ -5,6 +5,22 @@ const branch =
 
 const tokenForEveryBranch = "351c04276ed1059a64f12ee7ade4c2fab8e11562";
 
+const isValidVocaroURL = (link: string | undefined): boolean => {
+  if (!link) {
+    return true;
+  }
+
+  try {
+    const newUrl = new URL(link);
```

Comment:
```suggestion
    const newURL= new URL(link);
```

Just to keep the name convetion for this abbreviation in this file consistent